### ISSUE TITLE
The function spacemacs//restore-previous-display-config is missing and is breaking spacemacs-base.

### ISSUE THREAD

**Issue by magnetophon** [syl20bnr/spacemacs#6598]
#### Description :octocat:
develop branch breaks "restore-previous-display-config"

#### Reproduction guide :beetle:
```
rm .spacemacs
cd ~/emacs.d/
git checkout develop
git pull
```
- Start Emacs
- choose vim,base,helm
- ``<SPC>ff``
- choose file

*Observed behaviour:* :eyes: :broken_heart:
The helm buffer stays open, and no new file is opened.

*Expected behaviour:* :heart: :smile:
The helm buffer closes and a new file is opened.

#### Additional Info
- works on the ``master`` branch

#### System Info :computer:
- OS: gnu/linux
- Emacs: 24.5.2
- Spacemacs: 0.105.21
- Spacemacs branch: develop (rev. c64d04f)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp)
```

#### Backtrace :paw_prints:
```
Debugger entered--Lisp error: (void-function spacemacs//restore-previous-display-config)
  spacemacs//restore-previous-display-config()
  run-hooks(helm-cleanup-hook)
  helm-log-run-hook(helm-cleanup-hook)
  helm-cleanup()
  #[0 "\300 \207" [helm-cleanup] 1 "\n\n(fn)"]()
  funcall(#[0 "\300 \207" [helm-cleanup] 1 "\n\n(fn)"])
  #[0 "\311\211\211\211\305\206\n

**Comment by bmag** [comment]
What is "restore-previous-display-config"? Is it some configuration you've done? Do you mean the restoring of window configuration specifically at the end of helm sessions?

Also, I assume "choose file" means pressing `RET`?

I haven't tested the repro steps yet, will check it on Monday/Tuesday.

**Comment by magnetophon** [comment]
@bmag OK, thanks.

**Comment by magnetophon** [comment]
@bmag I really started missing ``develop`` :heart: so I did some more testing:
When you choose ``spacemacs`` instead of ``spacemacs-base`` the issue goes away.

**Comment by tuhdo** [comment]
The function `spacemacs//restore-previous-display-config` is missing and is breaking `spacemacs-base`.

**Comment by syl20bnr** [comment]
Anyone searched for the responsible commit ?

**Comment by bmag** [comment]
The function `spacemacs//restore-previous-display-config` is called from `helm/init-helm` in `helm` layer. The function is defined in file `layers/+spacemacs/spacemacs-completion/funcs.el`, but the layer `spacemacs-completion` is not loaded because it's not part of the distribution `spacemacs-base`. That causes the `void-function` error and the bug reported here.

By reading the code, the bug is at least from as far back as revision 1f993bd from two months ago. 

The fix is simple: don't call `spacemacs//restore-previous-display-config` from `helm` layer, instead move it to `spacemacs-completion` layer. A side effect of this move is that the helm window won't be forced to appear at the bottom of the frame with 0.3 of the frame's height, and instead it will be displayed according to other settings, maybe according to helm's default settings. I figured that's ok because `spacemacs-base` doesn't have `popwin` so it doesn't have any similar popup windows anyway. PR incoming.

**Comment by magnetophon** [comment]
@bmag thanks!

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
